packages/contract-method-name/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@cfxjs/contract-method-name",
+  "type": "module",
+  "main": "index.js",
+  "version": "0.0.2",
```

Comment:
don't alter the version manually, we are using the [defered versioning](https://yarnpkg.com/features/release-workflow#deferred-versioning) feature provided by yarn 2.0
I'll document that in clickup

Reply:
https://app.clickup.com/3719212/v/dc/3hg1c-6109/3hg1c-1720 @RickStormBringer

Reply:
Got it !